### PR TITLE
[ReadMe] updated readMe file for EavGraphQl-EncryptionKey modules

### PR DESCRIPTION
**Pull request by korovitskyi**
### Description (*)

Update README.md file for modules:
- Magento_EavGraphQl;
- Magento_Elasticsearch;
- Magento_Elasticsearch6;
- Magento_Elasticsearch7;
- Magento_Email;
- Magento_EncryptionKey.
### Fixed Issues (if relevant)

1. magento/devdocs magento/devdocs#4451: README content for modules - work in M2 Code

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by bgorski** [comment]
@magento run all tests

File: app/code/Magento/EavGraphQl/README.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
-# EavGraphQl
+# Magento_EavGraphQl module
 
-**EavGraphQl** primarily provides the GraphQl module information to generate metadata for Eav attributes.
+Magento_EavGraphQl module extends Magento_GraphQl and Magento_Eav modules to provides type and resolver information for  GraphQL API.
```
Comment (bgorski) [inline app/code/Magento/EavGraphQl/README.md]: Please fix double space before the "GraphQL" word and change "to provides" either to "and provides" or "to provide".

File: app/code/Magento/Elasticsearch/README.md [hunk 2]
```diff
@@ -1,2 +1,30 @@
-Magento\Elasticsearch module allows to use Elastic search engine for product searching capabilities.
-The module implements Magento\Search library interfaces.
\ No newline at end of file
+#Magento_Elasticsearch module
+
+Magento_Elasticsearch module allows using Elastic search engine for the product searching capabilities.
```
Comment (bgorski) [inline app/code/Magento/Elasticsearch/README.md]: Please rephrase to "using the Elasticsearch search engine", "Using the Elasticsearch engine" or something similar as the service is called Elasticsearch (not "Elastic" or "Elastic search").
Please also add a note that although it provides logic used by other modules implementing newer versions of Elasticsearch, this module by itself only adds support for Elasticsearch v5

File: app/code/Magento/EncryptionKey/README.md [hunk 3]
```diff
@@ -1 +1,9 @@
+#Magento_EncryptionKey module
```
Comment (bgorski) [inline app/code/Magento/EncryptionKey/README.md:1]: Please include the Extensibility section (https://github.com/magento/devdocs/wiki/Magento-module-README.md#extensibility) with the Layout subsection containing an information that this module introduces a new layout handle - `adminhtml_crypt_key_index`

**Review by bgorski — CHANGES_REQUESTED** [review]
Thank you for your contribution! I added a few comments to the newly introduced content, please review and make adjustments, but the overall effect looks good!

**Comment by korovitskyi** [comment]
Hi, @bgorski .
The changes were added.
Thank you for the review.

File: app/code/Magento/EavGraphQl/README.md [hunk 4]
```diff
@@ -1,3 +1,15 @@
-# EavGraphQl
+# Magento_EavGraphQl module
 
-**EavGraphQl** primarily provides the GraphQl module information to generate metadata for Eav attributes.
+Magento_EavGraphQl module extends Magento_GraphQl and Magento_Eav modules to provides type and resolver information for GraphQL API.
```
Comment (bgorski) [inline app/code/Magento/EavGraphQl/README.md:3]: There's still a typo in "provides" - should be "provide"

File: app/code/Magento/Elasticsearch/README.md [hunk 5]
```diff
@@ -1,2 +1,30 @@
-Magento\Elasticsearch module allows to use Elastic search engine for product searching capabilities.
-The module implements Magento\Search library interfaces.
\ No newline at end of file
+#Magento_Elasticsearch module
+
+Magento_Elasticsearch module allows using the Elasticsearch engine for the product searching capabilities.
```
Comment (bgorski) [inline app/code/Magento/Elasticsearch/README.md:3]: Please add a note that although it provides logic used by other modules implementing newer versions of Elasticsearch, this module by itself only adds support for Elasticsearch v5

**Review by bgorski — CHANGES_REQUESTED** [review]

**Comment by bgorski** [comment]
@korovitskyi you missed two things, though, so I re-added those parts of those comments. Please handle those as well. Thanks! We're getting closer!

**Comment by korovitskyi** [comment]
Hi, @bgorski .
This PR was reopened at https://github.com/magento/magento2/pull/33661.
I have trouble with the fork, so it was needed to recreate it.
Thank you for the review.

**Pull request closed without merging**